stdlib/public/core/HashedCollections.swift.gyb

```diff
@@ -2904,14 +2967,21 @@ internal struct _Native${Self}Storage<${TypeParameters}> {
   }
 %end
 
+  /// Wrap a fully initialized RawBuffer
```

Comment:
Please expand the [guidelines on doc comments](https://swift.org/documentation/api-design-guidelines/#write-doc-comment) and make these conform